File: API/Models/GestaoAdmAtivaModel.cs
namespace API_Visitatus.Models
{
    public class GestaoAdmAtivaModel
    {
        public long LojCodi { get; set; }        
        public string? LojNome { get; set; }
        public string? LojNumL { get; set;}
        public string? GstAdmNome { get; set;}
        public DateTime GstAdmDtIn { get; set;}
        public DateTime GstAdmDtFi { get; set;}
        public bool GstAdmStat { get; set;}
        public long CarCodi { get; set; }
        public string? CarNome { get; set;}
        public string? UsuNome { get; set; }
    }
}

File: Sistema/src/app/models/GestaoAdmAtiva.Model.ts
export class GestaoAdmAtivaModel {
  LojCodi: number;
  LojNome?: string;
  LojNumL?: string;
  GstAdmNome?: string;
  GstAdmDtIn: Date;
  GstAdmDtFi: Date;
  GstAdmStat: boolean;
  CarCodi?: number;
  CarNome?: string;
  UsuNome?: string;

  constructor(
    LojCodi: number,
    GstAdmDtIn: Date,
    GstAdmDtFi: Date,
    GstAdmStat: boolean,
    LojNome?: string,
    LojNumL?: string,
    GstAdmNome?: string,
    CarCodi?: number,
    CarNome?: string,
    UsuNome?: string
  ) {
    this.LojCodi = LojCodi;
    this.LojNome = LojNome;
    this.LojNumL = LojNumL;
    this.GstAdmNome = GstAdmNome;
    this.GstAdmDtIn = GstAdmDtIn;
    this.GstAdmDtFi = GstAdmDtFi;
    this.GstAdmStat = GstAdmStat;
    this.CarCodi = CarCodi;
    this.CarNome = CarNome;
    this.UsuNome = UsuNome;
  }
}
